api4/user_test.go

```diff
@@ -4529,6 +4529,36 @@ func TestPromoteGuestToUser(t *testing.T) {
 	})
 }
 
+func TestVerifyUser(t *testing.T) {
```

Comment:
Nonblocking comment, we should addapt the naming here as well:

```suggestion
func TestVerifyUserEmailWithoutToken(t *testing.T) {
```

Reply:
🤦‍♂️